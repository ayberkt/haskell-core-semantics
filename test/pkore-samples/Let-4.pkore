let(nonRec(tmVar([type omitted], name(f)),
           lam(tmVar([type omitted], name(x)), var(tmVar([type omitted], name(x))))),
    app(var(tmVar([type omitted], name(f))),
        lam(tmVar([type omitted], name(z)), var(tmVar([type omitted], name(z))))))
