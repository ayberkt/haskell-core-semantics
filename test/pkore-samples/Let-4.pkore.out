<generatedTop> <k> val ( 4 ) </k> <store> 1 |-> indirection ( 2 ) 2 |-> closure ( name ( x ) , var ( tmVar ( [type omitted] , name ( x ) ) ) , .Map ) 3 |-> indirection ( 4 ) 4 |-> closure ( name ( z ) , var ( tmVar ( [type omitted] , name ( z ) ) ) , name ( f ) |-> 2 ) </store> <env> .Map </env> <exit-code> 0 </exit-code> </generatedTop>
